tasks/acceptance-tests.yml

```diff
@@ -0,0 +1,65 @@
+---
+
+# acceptance-tests installer
+
+- name: "Checkout out acceptance tests repository"
+  git:
+    repo: "https://github.com/artefactual-labs/archivematica-acceptance-tests.git"
+    dest: "{{ archivematica_src_dir }}/archivematica-acceptance-tests"
+    version: "{{ archivematica_src_acceptance_tests_version }}"
+    force: "yes"
+
+- name: "Install pip dependencies in virtualenv"
+  pip:
+    chdir: "{{ archivematica_src_dir }}/archivematica-acceptance-tests"
+    requirements: "requirements.txt"
+    virtualenv: "/usr/share/python/acceptance-tests"
+    virtualenv_python: "python3.4"
+    state: latest
+
+- name: "Does the Google apt file exist?"
+  command: "test -f /etc/apt/sources.list.d/google-chrome.list"
+  register: google_apt_exists
+  ignore_errors: True
+
+- name: "Add Google Chrome key"
+  shell: "wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -"
+  when: google_apt_exists.rc == 1
+
+- name: "Add Google Chrome repo"
+  copy: content="deb http://dl.google.com/linux/chrome/deb/ stable main" dest=/etc/apt/sources.list.d/google-chrome.list owner=root group=root mode=644
+  when: google_apt_exists.rc == 1
+
+- name: Update apt cache
+  apt: update_cache=yes
+  when: google_apt_exists.rc == 1
+
+- name: Install packages
+  apt: pkg={{ item }} state=installed
+  with_items:
+    - xvfb
+    - google-chrome-stable
```

Comment:
This setup installs the current stable release of chrome, which right now is 58 on Ubuntu 14.04. I have had issues with getting all of the tests in features/core/ to run (and pass) with Chrome 58 and chromedriver 2.29. Have you been able to get these tests to pass with this setup?

I think the default should be Firefox v. 47. In my experience, the core/ tests pass consistently with Firefox v. 47 and Chrome 58 variably fails. In one attempt, I got `ConnectionRefusedError` in archivematicaselenium.py line 233 when `tear_down` tries to access the ``driver.window_handles`` property.